GoogleTestAdapter/Core/Runners/CommandLineGenerator.cs

```diff
@@ -59,6 +59,8 @@ public IEnumerable<Args> GetCommandLines()
             return commandLines;
         }
 
+        private Func<A, R> GetFuncWithAnonymousReturnType<A, R>(Func<A, R> f) { return f; }
```

Comment:
This is an interesting approach - have learned something here...

Due to the generality of this method, I suggest moving it to a new class `Common.Utils`.